src/Pages/SignUp/SignUp.js

```diff
@@ -1,8 +1,137 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 import './SignUp.scss';
 
 export default class SignUp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      pwd: '',
+      name: '',
+    };
+  }
+
+  clickHandler = () => {
+    //e.prevent.Default();
+    fetch('http://10.58.0.88:8000/user/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.pwd,
+        name: this.state.name,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        if (res.message === 'SUCCESS') {
+          this.props.history.push('/login');
+        }
+      });
+  };
+
+  //패스워드 유효성 검사
+  checkPwdHandler = (e) => {
+    e.preventDefault();
+    const checkPwd = function (str) {
+      var reg_pwd = /^.*(?=.{6,20})(?=.*[0-9])(?=.*[a-zA-Z]).*$/;
+      return !reg_pwd.test(str) ? false : true;
```

Comment:
```suggestion
      return reg_pwd.test(str) ? true : false;
```
이렇게 쓰는 건 어떨까요? 굳이 한번 뒤집어줄 이유가 없어보입니다.

Reply:
넵 수정하였습니다.